components/ambient-api-server/proto/ambient/v1/inbox.proto

```
@@ -0,0 +1,26 @@
+syntax = "proto3";
+
+package ambient.v1;
```

Buf (1.67.0)

[error] Files with package "ambient.v1" must be within a directory "ambient/v1" relative to root but were in directory "components/ambient-api-server/proto/ambient/v1".

(PACKAGE_DIRECTORY_MATCH)